shared/triage_metadata.go

```diff
@@ -0,0 +1,257 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//go:generate mockgen -destination sharedtest/triage_metadata_mock.go -package sharedtest github.com/web-platform-tests/wpt.fyi/shared TriageMetadataInterface
+
+package shared
+
+import (
+	"context"
+	"errors"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/google/go-github/v28/github"
+	"gopkg.in/yaml.v2"
+)
+
+// TriageMetadataInterface encapsulates triage() method for testing purpose.
```

Comment:
```suggestion
// TriageMetadataInterface encapsulates the Triage() method for testing.
```

(or 'for testing purposes')